src/examples/mfem_mortar_lm_patch.cpp

```diff
@@ -380,18 +380,15 @@ int main( int argc, char** argv )
   // Apply a restriction operator on the submesh: maps dofs stored in g to tdofs
```

Comment:
Does this comment need to change from using a restriction operator to the prolongation operator?